telegram/ext/jobqueue.py

```diff
@@ -56,6 +68,12 @@ def __init__(self, bot, prevent_autostart=None):
         """:type: float"""
         self._running = False
 
+    def dt_time_to_seconds(self, dt_time):
```

Comment:
Do you mean as method name, because `datetime.time` doesn't have a method called `total_seconds`. Am I missing something?